plugins/inputs/knx_listener/README.md

```diff
@@ -52,6 +52,10 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   #   addresses = ["5/5/3"]
 ```
 
+### Related tools
+
+- [knx-influx-config-generator](https://github.com/svsool/knx-influx-config-generator) generates configuration from KNX project file
```

Comment:
Hi,

Cool tool! I think we would consider adding this to the docs, but I have two questions/asks:

1) This project is called telegraf, not influx. Generating an "influx" config is a bit confusing as that isn't the project name or what we refer to the config file as.
2) Can you please put an open source licence in your project. It is your tool so you can pick any of them, but I would prefer we link to something that has one specified.

Thanks!

Reply:
hey @powersj, let me know if it's better now, thanks!

Reply:
perfect! thank you so much!